eng/common/scripts/Verify-ChangeLog.ps1

```diff
@@ -6,11 +6,12 @@ param (
   [string]$ServiceDirectory,
```

Comment:
Ditto here. I think you can remove these two files from your PR.

Reply:
This is also from commits to master. Should I still remove it from the pr?